mergify_engine/worker.py

```diff
@@ -112,31 +116,25 @@ async def push(redis, owner, repo, pull_number, event_type, data):
 
 
 async def get_pull_for_engine(owner, repo, pull_number, logger):
-    try:
-        async with await github.aget_client(owner, repo) as client:
-            try:
-                pull = await client.item(f"pulls/{pull_number}")
-            except http.HTTPNotFound:
-                # NOTE(sileht): Don't fail if we received even on repo/pull that doesn't exists anymore
-                logger.debug("pull request doesn't exists, skipping it")
-                return
+    async with await github.aget_client(owner, repo) as client:
```

Comment:
Are you sure we don't need to catch `MergifyNotInstalled` anymore? :(

Reply:
Yes, it shouldn't be here as it's caught later.

But, I see other kind of exceptions that should also be catched later like this one, that show up in Sentry while they should not.

I start thinking something isn't well catched somewhere in `_translate_exception_to_retries` or the `ThreadRunner`